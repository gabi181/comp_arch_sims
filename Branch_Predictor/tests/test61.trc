1 7 11 1 global_history global_tables using_share_mid
0x340943CC T 0x477871A4
0x340943CC T 0x477871A4
