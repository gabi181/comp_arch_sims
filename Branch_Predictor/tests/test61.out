0x340943cc N 0x340943d0
0x340943cc N 0x340943d0
flush_num: 2, br_num: 2, size: 305b
